integrations/openai/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@botpress/openai",
+  "integrationName": "openai",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start": "pnpm bp serve",
+    "build": "pnpm bp build",
+    "deploy": "pnpm bp deploy",
+    "type-check": "tsc --noEmit",
```

Comment:
**chore:** I use `type:check` in this repo

Reply:
Tbh I hate the inconsistency between our other repos and this one in terms of `type-check` and `type:check`. I think `type-check` makes more sense because `type:` is not really a namespace for multiple scripts. Feasible to do a search/replace in this repo in a separate PR to make this match the other repos?

Reply:
I don't like `type-check` as it doesn't fit my pattern:
```json
"scripts": {
    "type:check": "pnpm -r --stream type:check",
    "format:check": "prettier --check .",
    "format:fix": "prettier --write .",
    "lint:check": "eslint ./ --ext .ts --ext .tsx",
    "lint:fix": "eslint --fix ./ --ext .ts --ext .tsx",
    "dep:check": "depsynky check",
    "dep:fix": "depsynky sync",
    
    "check": "pnpm dep:check && pnpm format:check && pnpm lint:check && pnpm type:check",
    "fix": "pnpm dep:fix && pnpm format:fix && pnpm lint:fix"
  }
```